### ISSUE TITLE
Documentation  for scaling strategies in ScalingJobs incorrectly rules for the number of  job scale.

### ISSUE THREAD

**Issue by Shubham82** [kedacore/keda-docs#453]
In Scaling Jobs docs there is mentioned of different [scaling strategies](https://keda.sh/docs/2.2/concepts/scaling-jobs/#details) i.e default,custom,accurate. For every strategies there is also mentioned the rules for the number of job scale which is quite confusing i.e.
```
1) default
  queueLength - runningJobCount

2) custom
  queueLength - customScalingQueueLengthDeduction - (runningJobCount * customScalingRunningJobPercentage)

3) accurate
  if (maxScale + runningJobCount) > maxReplicaCount {
      return maxReplicaCount - runningJobCount
}
return queueLength
```
Here `queueLength`  creates confusion, Actually in the docs queueLength(Target Average value)  and Queue Length (length of the queue) are mentioned. These scaling strategies don't use queueLength and Queue Length directly  in the rules, it has used `maxScale`   whose value can be measured with this [rule](https://github.com/kedacore/keda/blob/347e8975d5f7b7ccc8e2bfc15311c47fd267707d/pkg/scaling/scale_handler.go#L290) i.e.

`maxValue = min(scaledJob.MaxReplicaCount(), devideWithCeil(queueLength, targetAverageValue))`

here  queuLength is the MetricName and its value can get from the GetMetrics()  function and  targetAverageValue is the  threshold or the number of messages that will be consumed on a job  which we mentioned in the trigger metadata,  its value can get from the GetMetricSpecForScaling() function.

So instead of queueLength we can use **maxScale** in the rules like used in the code also see [here](https://github.com/kedacore/keda/blob/347e8975d5f7b7ccc8e2bfc15311c47fd267707d/pkg/scaling/executor/scale_jobs.go#L314) and we can also add maxValue  rule in the docs for more clarity.

## Expected Behavior

The docs should according to the code or functionality.

## Actual Behavior

The docs not according to the code or functionality.

## Steps to Reproduce the Problem
No response

## Specifications

  - **KEDA Version:** *2.2.0*

**Comment by Shubham82** [comment]
Hi @tomkerkhove @zroubalik
could you please take a look!
if it is correct then i will raise a PR for it.

**Comment by tomkerkhove** [comment]
@TsuyoshiUshio Can you review the above please?

**Comment by Shubham82** [comment]
Thanks, @zroubalik @TsuyoshiUshio for your response!

I will soon create a PR for it.

**Comment by Shubham82** [comment]
Fixed it here #465
